framework/auth/cas.py

```diff
@@ -170,6 +170,16 @@ def revoke_application_tokens(self, client_id, client_secret):
         else:
             self._handle_error(resp)
 
+    def revoke_personal_token(self, token_id):
```

Comment:
To clarify understanding: I had assumed that CAS would find personal access tokens based on a database collection shared between the OSF and CAS. What is it about that mechanism that requires hitting the CAS revocation endpoint? (Why is removing it from the shared collection not enough?)